IonicPortals/IonicPortalsTests/PortalsPluginTests.swift

```diff
@@ -208,6 +208,21 @@ class PortalsPluginTests: XCTestCase {
         XCTAssertTrue(completed)
     }
     
+    func test_subscribeTo__does_not_fire_callback__when_cancellable_is_deallocated() {
```

Comment:
This test cropped up into this PR because I started going down a route of creating a type exposed to Objective-C that could use the `AnyCancellable` by wrapping it in an `@objc` class. I ended up not deciding not to do that, but left this here because it's useful insofar as making sure we don't break this down the line.